### ISSUE TITLE
add ability to filter by keywords in sender's name

### ISSUE THREAD

**Issue by mmomjian** [healthchecks/healthchecks#1084]
![image](https://github.com/user-attachments/assets/c5819925-89ea-4f42-b10b-cec0379e097a)

Certain services will include information in the sender's name. It would be nice to filter based on this field as well.

**Comment by mmomjian** [comment]
Correct. In Proxmox you can set the From name for each notification type, but the subject is auto set. In this way we can route notification of a critical severity to come from a different From name.

**Comment by webdev11193** [comment]
Can I work on this issue?
